server/command.go

```diff
@@ -252,12 +252,29 @@ func createSettingsCommand(optInstance bool) *model.AutocompleteData {
 		"list", "", "View your current settings")
 	settings.AddCommand(list)
 
-	notifications := model.NewAutocompleteData(
-		"notifications", "[on|off]", "Update your user notifications settings")
-	notifications.AddStaticListArgument("value", true, []model.AutocompleteListItem{
+	setting := []model.AutocompleteListItem{
 		{HelpText: "Turn notifications on", Item: "on"},
 		{HelpText: "Turn notifications off", Item: "off"},
-	})
+	}
+
+	notifications := model.NewAutocompleteData(
+		"notifications", "[assignee|mention|reporter|watching]", "manage notifications")
+
+	assigneeNotifications := model.NewAutocompleteData(assigneeRole, "", "manage assignee notifications")
+	assigneeNotifications.AddStaticListArgument("value", true, setting)
+
+	mentionNotifications := model.NewAutocompleteData(mentionRole, "", "manage mention notifications")
+	mentionNotifications.AddStaticListArgument("value", true, setting)
+
+	reporterNotifications := model.NewAutocompleteData(reporterRole, "", "manage reporter notifications")
+	reporterNotifications.AddStaticListArgument("value", true, setting)
+
+	watchingNotifications := model.NewAutocompleteData(watchingRole, "", "manage watching notifications")
+	reporterNotifications.AddStaticListArgument("value", true, setting)
```

Comment:
@Kshitij-Katiyar Can we also update the help Text in the file `server/plugin/template.go`. 

Currently the help text states:

```
/jira instance settings [setting] [value] - Update your user settings
[setting] can be notifications
[value] can be on or off
```

This might also need some update as we have now included additional options.

Reply:
@srkgupta I have updated the `commonHelpText` in `server/command.go`. 
Can you please elaborate what do you mean by 
> Can we also update the help Text in the file `server/plugin/template.go` ?

as we do not have any such file.